app/templates/frameworks/sign_framework_agreement.html

```diff
@@ -0,0 +1,57 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+    {{ framework.name }} Sign agreement – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+    {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Digital Marketplace",
+        "href": "/"
+      },
+      {
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
+      {
+        "text": framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
+      },
+      {
+        "text": "Sign agreement"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+
+{% block mainContent %}
+    {% include 'toolkit/forms/validation.html' %}
+    <span class="govuk-caption-xl">{{ framework.name }} {{ contract_title }}</span>
+    <h1 class="govuk-heading-xl">Supplier appointment terms</h1>
+    {% with company_details=company_details %}
+        {% include "frameworks/_framework_agreement_legal_text_{0}.html".format(framework_slug) %}
+        <p class="govuk-clearfix">
+            <a href="{{ framework_pdf_url }}">
+            <img style="float:left; border: 4px solid #ccc; margin-right: 10px;" alt="" src="{{ url_for('static', filename='images/pdf-thumbnails/{0}.png'.format(framework_slug)) }}">
```

Comment:
👀  @domoscargin I took this directly from the prototype, so there may be a better way to do it?

Reply:
I think a float in this case is fine. That's how it's handled in [publishing components](https://components.publishing.service.gov.uk/component-guide/attachment). 

Give the image a class - something like `app-file-image` and transfer the styles to SCSS (not sure if one of the existing scss files is suitable or if a new one has to be created). Use `$govuk-border-colour` instead of `#ccc`.

Looking at the publishing component, it might be good to add an `<abbr>` around PDF.